build.gradle

```diff
@@ -284,9 +284,11 @@ project('programming-extensions:programming-extension-vfsprovider') {
                 project(':programming-extensions:programming-extension-dataspaces-common')
         )
 
-        compile('activeeon:vfs-s3:2.4.8') {
-            exclude group: 'com.fasterxml.jackson.core'
-            exclude group: 'com.fasterxml.jackson.dataformat'
+        if (rootProject.hasProperty('s3')) {
```

Comment:
Would you add please a commented example about where and how to define the property 's3'.

Reply:
@nebilbenmabrouk done